### ISSUE TITLE
Scroll all views at once

### ISSUE THREAD

**Issue by hawkins** [kitze/sizzy#36]
Hey Kitze! I've been enjoying Sizzy a lot lately, but I find myself wishing I could scroll all views at once in order to see how the same portion of the page looks on multiple devices (similar to how Emmet Re:view scrolls). Is this a feature you've considered implementing at all?

While I'm new to Sizzy, I'd love to help out where I can and try implementing this myself if you feel this is a feature that may belong in the app. 😄

**Comment by kitze** [comment]
Hey @hawkins I was thinking about this and it kind of doesn't make sense. If you have an iPad in landscape on the left, and an iPhone 4 on the right, how should the scrolls be synced between the two? They will both be at different positions, so I don't see the point of syncing scroll if all the devices are in different sizes/orientations.

**Comment by polikin** [comment]
Hey @kitze, this is a nice feature when you want to see the portion of your page in different views at the same time. This is the same feature as Blisk.io or BrowserSync. It is very useful when you test your layout. It can be an option, like "Synchronizes scroll position" (checkbox) cc @hawkins

**Comment by kitze** [comment]
@polikin What I'm saying is, imagine this situation:

You have iPhone 4 which has 480px height and iPad which has 1024px height:

When the iPad is at 580px scroll, what is the scroll position on the iPhone?

Doesn't make sense, the scrolls will always be off for every device.

**Comment by hawkins** [comment]
Relative scrolling is how some other platforms handle that issue. In other words, when the user scrolls the iPad to 580px, the iPhone scroll can be calculated as something like `(580 / 1024) * 480` in units px.

You're right tho, this isn't useful for every site or layout (particularly those that change drastically on wider devices, such as iPad vs iPhone), so that's why I agree with @polikin that an option to enable this would probably be best.

**Comment by polikin** [comment]
In that situation, it doesn't make sense. But if you let the user choose which device to sync rather then scroll every device at the same time, maybe it will be better.

**Comment by progcode** [comment]
@hawkins @kitze you can do this w/ browsersync https://github.com/kitze/sizzy/issues/44

**Comment by saintplay** [comment]
Nice hack with browsersync

**Comment by lunelson** [comment]
Yeah FTR browsersync's implementation of scroll-synching is what I would have suggested for Sizzy, it's proportional WRT the height of each document

https://github.com/BrowserSync/browser-sync/blob/master/client/lib/ghostmode.scroll.js#L159-L167
